src/main/java/org/waarp/openr66/protocol/localhandler/LocalTransaction.java

```diff
@@ -314,20 +313,6 @@ public LocalChannelReference createNewClient(NetworkChannelReference networkChan
      * @return the LocalChannelReference
      */
     public LocalChannelReference getFromId(Integer id) {
-        int maxtry = (int) (Configuration.configuration.getTIMEOUTCON() / Configuration.RETRYINMS) / 2;
-        maxtry = 100;
-        for (int i = 0; i < maxtry; i++) {
-            LocalChannelReference lcr = localChannelHashMap.get(id);
-            if (lcr == null) {
-                try {
-                    Thread.sleep(Configuration.RETRYINMS);
-                    Thread.yield();
-                } catch (InterruptedException e) {
-                }
-            } else {
-                return lcr;
-            }
-        }
```

Comment:
I remember this code, when I noticed that sometime, there were a delay, without any explanation...
If there is no more issue, then you're right!!

Reply:
I tried every possible scenarii, and removing this loop does not present any issue.
However, in this case, when a transfer is stopped, it delays the processing of each packet by 1s from the moment that lcr is closed. It does no harm for a single transfer, but when several transfers between with the same server are stopped at the same time, the total processing delay can extend to several minutes... hence the removal!